dali/operators/expressions/arithmetic_meta.h

```diff
@@ -204,75 +176,136 @@ inline DALIDataType TypePromotion(span<DALIDataType> types) {
  */
 template <ArithmeticOp op, typename Backend>
 struct arithm_meta {
```

Comment:
shouldn't it be just forward declared (instead of empty)?

Reply:
Done